src/devices/Dhtxx/Dhtxx.csproj

```diff
@@ -22,7 +22,8 @@
       <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=linux</AdditionalProperties>
     </ProjectReference>
     <ProjectReference Include="..\Units\Units.csproj" />
+    <ProjectReference Include="..\Common\Iot\Device\Common\CommonHelpers.csproj" />
```

Comment:
Should this be in the sample project?

Reply:
I personally prefer it where it is, but will wait for your advice.

Reply:
You should only reference what you use. I don't think this uses it directly. Samples are

Reply:
Done (need to commit)